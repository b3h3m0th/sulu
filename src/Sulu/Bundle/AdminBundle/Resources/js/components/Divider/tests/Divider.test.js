/* eslint-disable flowtype/require-valid-file-annotation */
import {render} from 'enzyme';
import React from 'react';
import Divider from '../Divider';

test('Render an empty Divider', () => {
    const {container} = render(<Divider />);
    expect(container).toMatchSnapshot();
});

test('Render a Divider with text', () => {
    const {container} = render(<Divider>Test</Divider>);
    expect(container).toMatchSnapshot();
});
