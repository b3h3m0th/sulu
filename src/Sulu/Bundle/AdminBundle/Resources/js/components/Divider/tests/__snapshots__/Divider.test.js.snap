// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render a Divider with text 1`] = `undefined`;

exports[`Render an empty Divider 1`] = `undefined`;
